docs/calendar.md

```diff
@@ -0,0 +1,277 @@
+# Temporal.Calendar
+
+<details>
+  <summary><strong>Table of Contents</strong></summary>
+<!-- toc -->
+</details>
+
+A `Temporal.Calendar` is a representation of a calendar system.
+It includes information about how many days are in each year, how many months are in each year, how many days are in each month, and how to do arithmetic in that calendar system.
+
+Much of the world uses the [Gregorian calendar](https://en.wikipedia.org/wiki/Gregorian_calendar), which was invented in 1582 C.E.
+On the modern Internet, the most often used calendar system is the calendar standardized by ISO 8601, which is the same as the Gregorian calendar with the addition of week-numbering rules.
+In general it is extended backwards ("proleptically") to cover the period of history before its invention, which is an optional modification allowed by the ISO 8601 standard.
+
+However, the ISO 8601 calendar is not the only calendar in common use in the world.
+Some places use another calendar system as the main calendar, or have a separate calendar system as a commonly-used civil or religious calendar.
+
+> **NOTE:** The Temporal polyfill currently only includes the ISO 8601 calendar, but the Temporal proposal will eventually provide all the calendars supported by Intl.
```

Comment:
Can you ship the Japanese calendar in the polyfill since I think you already have it mostly done?

Reply:
I thought we had mentioned not doing that and instead encouraging others to implement Intl calendars in order to exercise the API?

I can add it, but I think it might need a bit more work than I will be able to complete this week if we still want to keep to our deadline of "within the next few days". On the other hand, it would certainly be nice to have an actual non-ISO calendar that we could use in the docs examples, so that we can make the docs examples actually executable. I'll see how far I get.